### PR TITLE
Adding liveness and readiness probe for the payload tracker consumer

### PR DESCRIPTION
**Pull request by tahmidefaz**
## What?
Adding two simple endpoint for the liveness and readiness probes

## Why?
needed by app-sre to check the health of the probes

## How?
added two endpoints to the metrics server that confirms the health of the consumer

## Testing
Did you add any tests for the change?

## Anything Else?
Any other notes about the PR that would be useful for the reviewer. 

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

File: deployments/clowdapp.yml [hunk 1]
```diff
@@ -82,6 +82,24 @@ objects:
         image: ${IMAGE}:${IMAGE_TAG}
         command: 
           - ./pt-consumer
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /live
+            port: 8001
```
Comment (dehort) [inline deployments/clowdapp.yml:89]: Is this the right port?  I _think_ the metrics port will get configured by clowder to be 9000?

File: deployments/clowdapp.yml [hunk 2]
```diff
@@ -82,6 +82,24 @@ objects:
...
+            scheme: HTTP
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 1
+        readinessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /ready
+            port: 8001
```
Comment (dehort) [inline deployments/clowdapp.yml:98]: Is this the right port?  I _think_ the metrics port will get configured by clowder to be 9000?

**Review by dehort — APPROVED** [review]
Looks good to me!

**Review by aleccohan — APPROVED** [review]
LGTM!!

**Pull request merged**